lib/components/src/toolbar/toolbar.ts

```diff
@@ -0,0 +1,57 @@
+import { customElement, property } from "lit/decorators.js";
+import styles from './toolbar.scss';
+import { TopAppBarBase } from '@material/mwc-top-app-bar/mwc-top-app-bar-base';
+import { html } from "lit";
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'td-toolbar': CovalentToolbarBase,
+  }
+}
+
+@customElement('td-toolbar')
+export class CovalentToolbarBase extends TopAppBarBase {
+  @property({type: Number}) breadcrumbs = 0;
+  
+  static override styles = [styles];
+  override render() {
+    // Create dummy data for the toolbar breadcrumbs.
+    let path: any[] = [];
+    if(this.breadcrumbs > 5) {
```

Comment:
`5` here seems to assume a large screen? (although, I often struggle with how to get this to work nicely on a variable width screen)

Reply:
At smaller screens, the breadcrumbs will just wrap but I don't know if that should be the intended behavior.

Reply:
They should truncate, with the second breadcrumb item being replaced with ..., then the third, etc. I believe you had an article @ecni19 illustrating this

Reply:
Something like `Section Title > ... > Page title` right?